typings/index.d.ts

```diff
@@ -24,6 +24,8 @@ declare module 'ytdl-core' {
       begin?: string | number | Date;
       liveBuffer?: number;
       highWaterMark?: number;
+      lang?: string;
```

Comment:
what's on with this?
didn't see any code related to that

Reply:
im pretty sure that was an option in ytdl before so it persisted here